fleet_adapter_template/fleet_adapter_template/fleet_adapter.py

```diff
@@ -208,6 +208,31 @@ def make_callbacks(self):
             )
         )
 
+        callbacks.localize = lambda estimate, execution: self.localize(
+            estimate, execution
+        )
+
+        return callbacks
+
+    def localize(self, estimate, execution):
+        self.node.get_logger().info(
+            f'Commanding [{self.name}] to change map to'
+            f' [{estimate.map}]'
+        )
+        if self.api.localise(self.name, estimate.position, estimate.map):
+            self.node.get_logger().info(
+                f'Localised [{self.name}] on {estimate.map} '
```

Comment:
`Localized`

Reply:
Addressed in [this commit](https://github.com/open-rmf/fleet_adapter_template/pull/41/commits/aa2e48d2a1654d600346c5c4e03c48170a93f443). Apologies for missing that.